config/environments/production.rb

```diff
@@ -63,6 +65,7 @@
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default_url_options = { :host => "https://multi-meet.herokuapp.com" }
```

Comment:
Thanks for figuring this out and making sure the code can seamlessly work for both dev environments and the actual environment!